### PR TITLE
Explore page

### PR DESCRIPTION
**Pull request by amartya-rs**
In this PR:
- explore page
- filter by category

files to review: [review me](https://github.com/amartya-rs/hhTube/pull/2/files/0fa8f97cb7c5ad90c82107ecbe02cddef9c561a3..1ef7ed1b041da98e307900810a0d16619fd5a552)

preview: https://deploy-preview-2--hh-tube.netlify.app/

### CODE REVIEW

File: src/components/video-card/VideoCard.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <div className="icon-wrapper">
+               <span>
+                  <AiOutlineLike size="24px" />
+               </span>
+               <span>
+                  <MdOutlineWatchLater size="24px" />
+               </span>
+               <span>
+                  <MdPlaylistAdd size="24px" />
+               </span>
```
Comment (itsmebipul00) [inline src/components/video-card/VideoCard.jsx]: Suggestion: You can make the sizes generic as youre repeating

File: src/pages/explore-page/ExplorePage.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                  >
+                     {ele.categoryName.toLowerCase()}
+                  </span>
+               ))}
+         </div>
+         <section className="video-section my-2">
+            {videosToDisplay &&
+               videosToDisplay.map((ele) => (
+                  <VideoCard key={ele._id} videoData={ele} />
+               ))}
```
Comment (itsmebipul00) [inline src/pages/explore-page/ExplorePage.jsx]: Suggestion:  optional chaning could be used

File: src/pages/home-page/HomePage.jsx [hunk 3]
```diff
@@ -14,13 +16,19 @@ const HomePage = () => {
...
          <section className="category-section my-4">
             {categories &&
-               categories.map((ele) => (
-                  <CategoryCard
-                     key={ele._id}
-                     categoryName={ele.categoryName}
-                     imageUrl={ele.imgUrl}
-                  />
-               ))}
+               categories
```
Comment (itsmebipul00) [inline src/pages/home-page/HomePage.jsx]: Suggestion:  optional chaing here too

**Review by itsmebipul00 — APPROVED** [review]
Good to merge !! Great work amartya!!

**Review by sudipkundu999 — APPROVED** [review]
The code looks good.
UI suggestion (need not follow): While hovering over the card then is a sudden increase in size if a slight animation could be given then it would feel better.

File: src/pages/explore-page/ExplorePage.jsx (hunk 2 above)
Comment (amartya-rs) [reply src/pages/explore-page/ExplorePage.jsx]: Done.

File: src/pages/home-page/HomePage.jsx (hunk 3 above)
Comment (amartya-rs) [reply src/pages/home-page/HomePage.jsx]: Done.

File: src/components/video-card/VideoCard.jsx (hunk 1 above)
Comment (amartya-rs) [reply src/components/video-card/VideoCard.jsx]: Created a variable for them. Also, I have a default size but here in card need small icons, so used size prop here to override the default one.

**Pull request merged**